bin/cliain/rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2023-01-10"
+channel = "nightly-2023-05-22"
```

Comment:
Is it possible to use some stable version? Like 1.70.X ?

Reply:
nope, aleph-client uses features from nightly only